web_src/js/features/user-block.js

```diff
@@ -0,0 +1,25 @@
+import $ from 'jquery';
+
+export function initUserBlock() {
+  $(document).on('click', '.block-user-action', function (event) {
```

Comment:
I guess it doesn't need to write JS code.

Maybe the existing `show-modal` already covers this case.

![Uploading image.png…]()

Reply:
Cool, didn't know we have this method. While looking for an example I found
https://github.com/go-gitea/gitea/blob/a9bc590d5d10b97bd8aa050ffb720e141a600064/templates/repo/branch/list.tmpl#L54-L61
but that's a bad example because there are two click handlers if I'm not mistaken?
https://github.com/go-gitea/gitea/blob/a9bc590d5d10b97bd8aa050ffb720e141a600064/web_src/js/features/repo-branch.js#L27-L42

Reply:
Thank you, moved to `show-modal`.